package.yaml

```diff
@@ -0,0 +1,70 @@
+name:                servant-elm-street
+version:             0.1.0.0
+github:              "githubuser/servant-elm-street"
+license:             BSD3
+author:              "Author name here"
+maintainer:          "example@example.com"
+copyright:           "2021 Author name here"
+
+extra-source-files:
+- README.md
+- ChangeLog.md
+
+# Metadata used when publishing your package
+# synopsis:            Short description of your package
+# category:            Web
+
+# To avoid duplicated efforts in documentation and dealing with the
+# complications of embedding Haddock markup inside cabal files, it is
+# common to point users to the README.md file.
+description:         Please see the README on GitHub at <https://github.com/githubuser/servant-elm-street#readme>
+
+dependencies:
+- base >= 4.7 && < 5
+
+library:
+  source-dirs: src
+
+dependencies:
+- servant-foreign
+- text
+- servant-elm
+- base >= 4.7 && < 5
+- aeson
+- casing
+- elm-street
+- filepath
+- servant
+- servant-server
+- wai
+- warp
+- servant-js
+- random
+- elm-bridge
+- wl-pprint-text
+- microlens
+
```

Comment:
We should turn on more warnings and error out on warnings. Here's a snippet to put in this file:
```yaml
ghc-options:
- -Wall
- -Werror
- -Wincomplete-uni-patterns
- -Wredundant-constraints
- -Widentities
- -Wincomplete-record-updates
- -Wmissing-deriving-strategies
- -Wmissing-export-lists
```

Reply:
done